### PR TITLE
Ck 124 add animation to front end

### PR DESCRIPTION
**Pull request by Tosvng**
Second to the last commit was on the wrong branch

### CODE REVIEW

**Review by JoshuaDueck — APPROVED** [review]
Conditional approval, code looks fine. Would be nice to have some gifs, but I can also pull and try it out. Not sure why sonarcloud is failing?

**Review by JoshuaDueck — CHANGES_REQUESTED** [review]

File: frontend/src/components/pages/ChoresPage.js [hunk 1]
```diff
@@ -302,19 +304,31 @@ const ChoresPage = ({ isChoresLoading, choresData }) => {
...
+                    <motion.tr
+                      {...row.getRowProps()}
+                      initial={{
+                        opacity: 0,
+                        translateY: -50,
+                      }}
+                      animate={{
+                        opacity: 1,
+                        translateY: 0,
+                        transition: { duration: 0.3, delay: i * 0.2 },
```
[the remark this answers is not in the document]
Comment (Tosvng) [reply frontend/src/components/pages/ChoresPage.js:316]: okay

**Review by JoshuaDueck — APPROVED** [review]
I like the changes you made, thanks!

I still feel like the table animation could be removed (because navigating with pagination re-animates all the items so often), but if everyone else is a fan then that's alright. I approve.

**Review by Amikz — APPROVED** [review]
This looks great! Nice job!

**Review by Ahsen19 — APPROVED** [review]

File: .github/workflows/sonarcloud.yml
Comment (Ahsen19) [inline .github/workflows/sonarcloud.yml:1]: No need for this anymore

**Pull request merged**